components/Alert.tsx

```diff
@@ -0,0 +1,46 @@
+import clsx from 'clsx'
+import { DetailedHTMLProps, HTMLAttributes, ReactNode, useState } from 'react'
+import { FaExclamationTriangle, FaInfoCircle, FaTimes } from 'react-icons/fa'
+import { IconType } from 'react-icons/lib'
+
+type BaseProps<T = HTMLDivElement> = DetailedHTMLProps<HTMLAttributes<T>, T>
+
+export type AlertType = 'info' | 'warning' | 'error' | 'ghost'
+
+const ALERT_ICONS_MAP: Record<AlertType, IconType | null> = {
+  info: FaInfoCircle,
+  warning: FaExclamationTriangle,
+  error: FaTimes,
+  ghost: null,
+}
+
+export interface AlertProps extends BaseProps {
+  type?: AlertType
+}
+
+const Alert = (props: AlertProps) => {
+  const { type = 'info', className, children, ...rest } = props
+  const AlertIcon = ALERT_ICONS_MAP[type]
+
+  return (
+    <div
+      className={clsx(
+        'flex relative p-4 space-x-4 border-l-4',
+        { 'bg-blue-500/25 border-blue-500': type == 'info' },
+        { 'bg-yellow-500/25 border-yellow-500': type == 'warning' },
+        { 'bg-red-500/25 border-red-500': type == 'error' },
+        { 'bg-stone-500/25 border-stone-500': type == 'ghost' },
+        className
+      )}
+      {...rest}
+    >
+      {AlertIcon && <AlertIcon size={24} />}
+      <div className="flex flex-col flex-grow space-y-2">
+        {children}
+        {/*  */}
+      </div>
+    </div>
+  )
+}
+
+export default Alert
```

Comment:
I have read that it's better to use named exports for better tree-shaking. Thoughts?

Reply:
Yes it does. We could have a separate PR converting conventions and tidy up DX related quirks.

Reply:
Separate PR is good 👍 Can you please make an issue for that?

Reply:
https://github.com/CosmosContracts/juno-tools/issues/92